evalml/tests/component_tests/test_utils.py

```diff
@@ -35,8 +35,12 @@ def test_all_components(
         n_components = 41
     elif is_using_conda:
         n_components = 52
-    elif is_running_py_39_or_above or is_using_windows:
+    elif is_using_windows and not is_running_py_39_or_above:
```

Comment:
There was a slight bug in the prophet pr I think. This is what the number of components were before prophet and the new number of components depending on the environment:

![image](https://user-images.githubusercontent.com/41651716/128394431-179ca466-5ad5-4b98-93c8-bc87c1edb978.png)